packages/components/src/components/vertical-tab/vertical-tab.scss

```diff
@@ -175,7 +175,7 @@ $MIN_HEIGHT_FLOATING: calc(
             padding: 0 0.4rem;
             margin: 0 0.4rem;
             background-color: var(--text-loss-danger);
-            color: var(--general-main-1);
+            color: $COLOR_WHITE;
```

Comment:
Why reverting from a CSS variable to sass variable? is this against our styling conventions?

Reply:
Because this text doesn't need theming, it should remain as white color for both light and dark theme.

Reply:
you are right. But we have a `--status-colored-background` color that is white in both themes.

Reply:
ahh you're right, didn't know this sass variable